repose-aggregator/components/filters/cors-filter/src/main/scala/org/openrepose/filters/cors/CorsFilter.scala

```diff
@@ -61,6 +61,15 @@ class CorsFilter @Inject()(configurationService: ConfigurationService)
   }
 
   override def doFilter(servletRequest: ServletRequest, servletResponse: ServletResponse, filterChain: FilterChain): Unit = {
+    if (!isInitialized) {
+      logger.error("Filter has not yet initialized... Please check your configuration files and your artifacts directory.")
+      servletResponse.asInstanceOf[HttpServletResponse].sendError(500)
+    } else {
+      doFilterGuarded(servletRequest, servletResponse, filterChain)
```

Comment:
I totally see what you're saying.  I'll take the bullet, though.  I'd rather go through some difficulty now then wonder why we did that each time I run into this code.